Src/Base/AMReX_MultiFabUtil.H

```diff
@@ -32,12 +32,18 @@ namespace amrex
     void average_face_to_cellcenter (MultiFab& cc, int dcomp,
                                      const Vector<const MultiFab*>& fc,
                                      int ngrow = 0);
+    //! Average face-based MultiFab onto cell-centered MultiFab.
+    void average_face_to_cellcenter (MultiFab& cc, int dcomp,
+                                     const Vector<const MultiFab*>& fc,
+                                     IntVect ngrow = IntVect::TheZeroVector());
```

Comment:
```suggestion
                                     IntVect const& ngrow);
```

Reply:
This cannot be an optional argument because the int version is already optional.